test/fonts/test_add_font.py

```diff
@@ -60,8 +60,8 @@ def test_deprecation_warning_for_FPDF_CACHE_DIR():
 def test_add_font_unicode_with_path_fname_ok(tmp_path):
     for font_cache_dir in (True, tmp_path, None):
         pdf = FPDF(font_cache_dir=font_cache_dir)
-        font_file_path = HERE / "../fonts/Roboto-Regular.ttf"
-        pdf.add_font("Roboto-Regular", fname=font_file_path, uni=True)
+        font_file_path = HERE / "Roboto-Regular.ttf"
+        pdf.add_font("Roboto-Regular", fname=str(font_file_path), uni=True)
```

Comment:
Could this conversion to `str` be removed?
The code should be made to support `Path`-like `fname` parameter values

Reply:
Oops, sorry, I though this change was suggested by @alexp1917 ("_first, i found that we expect `font_file_path` to be a string and not a `pathlib.Path`_"). Seems to be a matter of taste. But yeah, the conversion could be removed.

Reply:
No, I understand it now, it's perfectly fine like this.
Merging this PR now